composer.json

```diff
@@ -26,9 +26,9 @@
     },
     "require": {
         "php": "^7.2",
+        "dragonmantank/cron-expression": "^2.3",
```

Comment:
Could you lower minimum required version to `2.0`? Maybe, in the wild, someone can't update this package to `2.3` version for some reason and `Crunz` doesn't really need newest version.

Reply:
Done